server/api.ts

```diff
@@ -124,7 +132,10 @@ const parseWarmup = async (warmup: string, mod: WarmupMod, tourney: string) => {
     throw new Error("Warmup map too long (max 3 minutes)");
   }
 
-  return `https://osu.ppy.sh/beatmapsets/${mapData.beatmapSetId}#taiko/${warmupMapId}`;
+  return `https://osu.ppy.sh/beatmapsets/${mapData.beatmapSetId}#${
+    // GameMode defines catch but in beatmap url it's fruits
+    tourneyData.mode == "catch" ? "fruits" : tourneyData.mode
```

Comment:
```suggestion
    tourneyData.mode === "catch" ? "fruits" : tourneyData.mode
```
for consistency (with the rest of this file, at least)

Reply:
Applied in d4c25040276da79ecc5d20a5ff4b2d3a4cbaa008, though it's moved to `utils.ts`